src/nvim/main.c

```diff
@@ -341,6 +341,17 @@ int main(int argc, char **argv)
   source_startup_scripts(&params);
 
   /*
+   * If using the runtime (-u is not NONE),
+   * enable syntax and filetype plugins
+   */
+  if (params.use_vimrc != NULL && strcmp(params.use_vimrc, "NONE") != 0) {
```

Comment:
I don't think we need to worry about it in this case: [are-char-argv-arguments-in-main-null-terminated](https://stackoverflow.com/questions/11020172/are-char-argv-arguments-in-main-null-terminated)

(here's the standard, which seems to match up: http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1256.pdf)